e2etests/timestamp_test.go

```diff
@@ -0,0 +1,45 @@
+package e2etests
+
+import (
+	"testing"
+	"time"
+
+	connector "database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	sqle "github.com/dolthub/go-mysql-server"
+	"github.com/dolthub/go-mysql-server/memory"
+	"github.com/dolthub/go-mysql-server/server"
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/analyzer"
+	"github.com/stretchr/testify/require"
+)
```

Comment:
This ordering should keep the `Check/Format PR Verify format` GitHub action happy... 

```suggestion
import (
	connector "database/sql"
	"testing"
	"time"

	_ "github.com/go-sql-driver/mysql"
	"github.com/stretchr/testify/require"

	sqle "github.com/dolthub/go-mysql-server"
	"github.com/dolthub/go-mysql-server/memory"
	"github.com/dolthub/go-mysql-server/server"
	"github.com/dolthub/go-mysql-server/sql"
	"github.com/dolthub/go-mysql-server/sql/analyzer"
)
```